easybuild/easyconfigs/m/MIRA/MIRA-4.0.2-intel-2017b.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'MIRA'
+version = '4.0.2'
+
+homepage = 'https://sourceforge.net/projects/mira-assembler/'
+description = """MIRA is a whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina),
+ IonTorrent data and PacBio (the later at the moment only CCS and error-corrected CLR reads)."""
```

Comment:
@boegel - I know you've copied and pasted here, but surely this should be `the latter`.